src/injected/element-finder-by-path.ts

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import Q from '../../node_modules/@types/q';
```

Comment:
Is this intentional? This doesn't look correct. Can you please check?

Reply:
Q officially removed! Thank you